raui-core/src/widget/mod.rs

```diff
@@ -570,18 +633,16 @@ macro_rules! widget {
             $crate::widget::node::WidgetNode::Component(component)
         }
     };
-}
-
-#[macro_export]
-macro_rules! widget_wrap {
-    ({$expr:expr}) => {
+    (@wrap {$expr:expr}) => {
         $crate::widget::node::WidgetNode::from($expr)
     };
-    ($tree:tt) => {
+    (@wrap $tree:tt) => {
         $crate::widget!($tree)
     };
 }
 
+/// Helper to destructure a struct on one line
+#[deprecated = "This macro is unused and will be removed soon"]
 #[macro_export]
```

Comment:
This macro is unused so I marked it deprecated. Tell me if we should do something else.

Reply:
it is unused and it will never be - we should totally remove it